packages/material-ui-lab/src/MonthPicker/PickersMonth.tsx

```diff
@@ -1,63 +1,93 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import Typography from '@material-ui/core/Typography';
-import { MuiStyles, StyleRules, WithStyles, withStyles } from '@material-ui/core/styles';
+import Typography, { TypographyTypeMap } from '@material-ui/core/Typography';
+import {
+  experimentalStyled,
+  unstable_useThemeProps as useThemeProps,
+} from '@material-ui/core/styles';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
 import { onSpaceOrEnter } from '../internal/pickers/utils';
+import pickersMonthClasses, { getPickersMonthUtilityClass } from './pickersMonthClasses';
```

Comment:
Can we inline this file instead?

Reply:
@mnajdova should we inline the ...Classes inside the component file for internal component

Reply:
Yeah, makes sense 👍